storage-manager/src/Ownership.cpp

```diff
@@ -171,10 +181,23 @@ void Ownership::releaseOwnership(const bf::path& p, bool isDtor)
 
   if (isDtor)
   {
-    // This is a quick release.  If this is being destroyed, then it is through the graceful
-    // shutdown mechanism, which will flush data separately.
-    DELETE(p, "OWNED");
-    DELETE(p, "FLUSHING");
+    {
```

Comment:
The pattern of this scope looks similar to another one both used later in this func and in `_takeOwnership`. Can we replace the boilerplate code with a func?

Reply:
make sense.